assets/css/calypsoify.css

```diff
@@ -2042,6 +2042,7 @@ form[name="cleanup_options"] fieldset label select {
 	color: #00a0d2;
 }
 
+/* TODO remove after https://github.com/Automattic/jetpack/pull/13126 is released */
```

Comment:
Nitpick: Can update these to `@todo` as well.

Reply:
Can you provide me with the code/instructions for adding that notice? There may be one or more classes of notice missed in this.

Reply:
Sure!  If you install and activate Shipstation ( https://woocommerce.com/products/shipstation-integration/ ) the notice should be added to those pages without any additional configuration.  If you've previously installed Shipstation, you may need to dump its options from `wp_options` or use a fresh install.

Reply:
fixed in https://github.com/Automattic/wc-calypso-bridge/pull/491/commits/70e7e1e94264fb6553d259b0eac830c8c71a98b6